src/form-field/interfaces.ts

```diff
@@ -33,6 +33,11 @@ export interface FormFieldProps extends BaseComponentProps {
    */
   label?: React.ReactNode;
 
+  /**
+   * An object containing all the necessary localized strings required by the component.
```

Comment:
"An object containing all localized strings required by the component."

Reply:
This is copied verbatim from [other instances of the i18n prop](https://polaris.a2z.com/components/date-range-picker/?tabId=api). Are you suggesting a global change here? If so I think we would want to do that here then the rest in another PR